EKrumynas/Migrations/20220505075202_BlogPostAddedCreatedAt.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace EKrumynas.Migrations
+{
+    public partial class BlogPostAddedCreatedAt : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<DateTime>(
+                name: "CreatedAt",
+                table: "BlogPosts",
+                type: "datetime2",
+                nullable: false,
+                defaultValue: new DateTime(1, 1, 1, 0, 0, 0, 0, DateTimeKind.Unspecified));
```

Comment:
DateTime.UtcNow; ? We will need to save default values in UTC. Will this work? Ofcourse we will be able to set it in code, but this way it seems that DB will have this random value?

Reply:
Not sure what you mean here but this part of code is generated by entity framework.